tests/lexer/lexer_test.cpp

```diff
@@ -1,13 +1,122 @@
 #include "catch2/catch.hpp"
 #include "lexer.hpp"
+#include "token.hpp"
+#include <string>
+#include "fmt/format.h"
+#include "trie.hpp"
+#include "catch_helpers.hpp"
+#include <vector>
+#include <iostream>
 
-SCENARIO("Summing two numbers") {
-    // By the way, don't mention constants in descriptions
-    GIVEN("Two positive numbers: 2 and 3") {
-        int a = 2, b = 3;
+namespace testing
+{
 
-        THEN("their sum is 5") {
-            REQUIRE(sum(a, b) == 5);
+    void checkInTrie(std::string key, common::Trie<TokenType>
+                                          trie)
```

Comment:
What the hell happened here? :)
```suggestion
    void checkInTrie(std::string key, common::Trie<TokenType> trie)
```